### ISSUE TITLE
It does not work for my project.

### ISSUE THREAD

**Issue by Gcaufy** [andrezero/boilerplate-esm-nyc-mocha#2]
I have tried this boilerplate, it works quite well.

but I tried to copy my project code here, it does not work as what I expected.

and also I tried to copy your test code to my project. it does not work either.

have no idea why it does not work.

it always shows:

```
Statements   : Unknown% ( 0/0 )
Branches     : Unknown% ( 0/0 )
Functions    : Unknown% ( 0/0 )
Lines        : Unknown% ( 0/0 )
```

It would be very helpful if you have any experiences on that.

**Comment by andrezero** [comment]
@Gcaufy I am so sorry I caused some frustration here. 

I believe this PR would have addressed the issue https://github.com/andrezero/boilerplate-esm-nyc-mocha/pull/1 but it was closed probably because I never attended to it. 

If I don't get this fixed, I promise to update the README to warn other people that the boilerplate is not actively maintained has known issues

**Comment by renie** [comment]
Should I be quoted here @andrezero ? Sorry, I don't really remember anything related to that.

**Comment by andrezero** [comment]
sorry, mistake. removed, feel free to undo your comment

**Comment by TobiTenno** [comment]
I've also not been able to [get it to work](https://github.com/TobiTenno/i18n-string-templates). I've been trying to sort it out with either nyc or c8, which seems to work, but not work with my IDE, which is kind of ideal.

**Comment by andrezero** [comment]
Hi @TobiTenno sorry to hear it's not working for you.

Have you considered building it from scratch? It's been 4 years since this boilerplate, and a lot things have evolved, namely [Node.js support for ESM](https://nodejs.org/api/esm.html) 

> Node.js treats JavaScript code as CommonJS modules by default. Authors can tell Node.js to treat JavaScript code as ECMAScript modules via the .mjs file extension, the package.json "type" field, or the --input-type flag. See Modules: Packages for more details.

**Comment by TobiTenno** [comment]
> Hi @TobiTenno sorry to hear it's not working for you.
> 
> Have you considered building it from scratch? It's been 4 years since this boilerplate, and a lot things have evolved, namely [Node.js support for ESM](https://nodejs.org/api/esm.html) 

Yes, I did it from scratch. I can across this project when trying to get nyc working with ESM.
